src/components/ProfilePage/LevelBox/LevelButton.tsx

```diff
@@ -0,0 +1,36 @@
+import { Button } from '../../../components/Button';
+import { NoSelectTypography } from '../ProfilePage.style';
+import { XPIcon } from '../../illustrations/XPIcon';
+
+interface LevelButtonProps {
+  level: number;
+  points: number;
+}
+
+export const LevelButton = ({ level, points }: LevelButtonProps) => {
+  return (
+    <Button
+      aria-label="XP Level"
+      variant="secondary"
+      size="medium"
+      styles={{
+        display: 'flex',
+        justifyContent: 'center',
+        alignItems: 'center',
+        pointerEvents: 'none',
+        paddingLeft: '12px',
+        height: '32px',
+      }}
+    >
```

Comment:
I know it´s shitty, but this Button component is not adding any value and I see it as deprecated. 

I´d suggest only directly using the styled component here called "ButtonSecondary" until we make the move to put these Buttons right into our theme MuiButtons (creating our custom variants).